profiles/common/modules/custom/nexteuropa_piwik/nexteuropa_piwik.install

```diff
@@ -91,4 +91,111 @@ function nexteuropa_piwik_enable() {
   );
   // Grant the permissions to PIWIK administrator role.
   user_role_change_permissions($piwik_admin_role->rid, $piwik_admin_permissions);
+
+  $administrator = user_role_load_by_name('administrator');
+
+  if ($administrator) {
+    user_role_grant_permissions(
+      $administrator->rid,
+      array(
+        'administer nexteuropa_piwik',
+      )
+    );
+  }
+}
+
+/**
+ * Implements hook_schema().
+ */
+function nexteuropa_piwik_schema() {
+  $schema = array();
+
+  $schema['nexteuropa_piwik_rule'] = array(
```

Comment:
This array is the same that used here: https://github.com/ec-europa/platform-dev/pull/1259/files#diff-7ce62a7971f302457a6782b94566fac3R156
Maybe, we can create a custom function to return this array.